tests/common/dualtor/tunnel_traffic_utils.py

```diff
@@ -95,7 +102,8 @@ def _build_tunnel_packet(outer_src_ip, outer_dst_ip):
             exp_pkt.set_do_not_care_scapy(IP, "ttl")
             exp_pkt.set_do_not_care_scapy(IP, "proto")
             exp_pkt.set_do_not_care_scapy(IP, "chksum")
-            exp_pkt.set_ignore_extra_bytes()
+            if inner_packet is None:
+                exp_pkt.set_ignore_extra_bytes()
```

Comment:
Do we need to set the don't care fields for inner packet?

Reply:
If we specify `inner_packet`, it is expected to be matched(extra bits should not be ignored). So for `test_orchagent_slb`, it will try to match those send packets in the tunnel traffic instead of those BGP keepalive packets.